tools/hyperscan-toolchain.sh

```diff
@@ -21,7 +20,8 @@ echo "Downloading binutils $BINUTILS_VERSION..."
 curl -C - --progress-bar "https://ftp.gnu.org/gnu/binutils/binutils-$BINUTILS_VERSION.tar.bz2" -o "binutils-$BINUTILS_VERSION.tar.bz2"
 
 echo "Downloading gcc $GCC_VERSION..."
-curl -C - --progress-bar "https://ftp.gnu.org/gnu/gcc/gcc-$GCC_VERSION/gcc-$GCC_VERSION.tar.bz2" -o "gcc-$GCC_VERSION.tar.bz2"
+git clone --depth=1 git://gcc.gnu.org/git/gcc.git gcc
```

Comment:
can we make this fixed to a specific version? I prefer working off of releases instead of the git repo.

Reply:
the patch works for that specific commit :(